src/components/Attachments/AttachmentView/AttachmentViewPdf/index.android.js

```diff
@@ -22,52 +23,73 @@ function AttachmentViewPdf(props) {
     // frozen, which combined with Reanimated using strict mode since 3.6.0 was resulting in errors.
     // Without strict mode, it would just silently fail.
     // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze#description
-    const shouldPagerScroll = attachmentCarouselPagerContext !== null ? attachmentCarouselPagerContext.shouldPagerScroll : undefined;
+    const isScrollEnabled = attachmentCarouselPagerContext === null ? undefined : attachmentCarouselPagerContext.isScrollEnabled;
+
+    const offsetX = useSharedValue(0);
+    const offsetY = useSharedValue(0);
+    const isPanRunning = useSharedValue(false);
```

Comment:
Can you explain what `isPanRunning ` means? I don't know exactly what is referring to.

Reply:
i renamed it to `isPanGesutureActive`. i think that's more self-explanatory